Dockerfile.windows

```diff
@@ -101,6 +104,24 @@ RUN cmake --build . --config Release; `
 
 # TODO: Do something with /work/out/bin/fluent-bit.{exe,dll}
 
+
+###############################################################################
+# Build JMX Project
+###############################################################################
+
+FROM base as javacontrib
+
+ADD https://javadl.oracle.com/webapps/download/AutoDL?BundleId=242044_3d5a2bb8f8d4428bbe94aed7ec7ae784 /local/jdk-8u251-windows-x64.exe
+RUN powershell start-process -filepath C:\local\jdk-8u251-windows-x64.exe -passthru -wait -argumentlist '/s,INSTALLDIR=C:\Java,/L,install64.log'
+
+COPY submodules/opentelemetry-java-contrib /work/submodules/opentelemetry-java-contrib
+
+WORKDIR /work/submodules/opentelemetry-java-contrib
+
+RUN ./gradlew --no-daemon :jmx-metrics:build
+# TODO: Parameterize this jar name once we can control the release artifact
+RUN Copy-Item -Path contrib/jmx-metrics/build/libs/opentelemetry-java-contrib-jmx-metrics-1.0.0-alpha.jar -Destination /work/out/bin/;
```

Comment:
Would you mind renaming this JAR file to not include the version?

- The version should be encoded inside the JAR
- Ideally we don't have to copy-paste the JAR name all over the build every time we bump its version.
- If we upgrade the ops-agent in-place on a machine, we should be replacing the JAR file instead of writing a new one.